src/test/java/juja/sqlcmd/ApplicationTest.java

```diff
@@ -1,10 +1,55 @@
 package juja.sqlcmd;
 
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.sql.SQLException;
+
+import static org.junit.Assert.assertEquals;
+
 public class ApplicationTest {
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+
+    private final PrintStream originalOut = System.out;
+    private final PrintStream originalErr = System.err;
+
+    @Before
+    public void setUpStreams() throws SQLException {
+        System.setOut(new PrintStream(outContent));
+        System.setErr(new PrintStream(errContent));
+    }
+
+    @After
+    public void cleanUpStreams() {
+        System.setOut(originalOut);
+        System.setErr(originalErr);
+    }
+
     @Test
     public void testloadApplication() {
-        new Application();
+        try {
+            new Application().simpleSQL();
```

Comment:
This test wouldn't pass on Windows machine because of the different line separator. You could use System.lineSeparator() to make your code cross-platform

Reply:
I wrote that issue before seen your "#22: Refactored. Use System.lineSeparator();" commit